### DISCUSSION TITLE
macos M1 qemu-img:  cannot execute binary file

### DISCUSSION THREAD

**Discussion by mbbee** [utmapp/UTM#4809] [UTM Help (macOS)]
What do i miss? Or is this a bug?

>>/Applications/UTM.app/Contents/Frameworks/qemu-img.framework/Versions/A/qemu-img 
-bash: /Applications/UTM.app/Contents/Frameworks/qemu-img.framework/Versions/A/qemu-img: cannot execute binary file

>> file qemu-img 
qemu-img: Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit dynamically linked shared library x86_64] [arm64]
qemu-img (for architecture x86_64):	Mach-O 64-bit dynamically linked shared library x86_64
qemu-img (for architecture arm64):	Mach-O 64-bit dynamically linked shared library arm64

**Comment by ktprograms** [comment]
The qemu binaries included in UTM are shared libraries as that's how UTM uses them. To get executable binaries, install the `qemu` package from Homebrew.
